src/markdown_generators.js

```diff
@@ -77,8 +77,15 @@ const generateMarkdownRecursive = (page, courseUidsLookup, courseData) => {
   const hasParent = parents.length > 0
   const parent = hasParent ? parents[0] : null
   const inRootNav = page["parent_uid"] === courseData["uid"]
+  const isInstructorInsightsSection =
+    page["short_url"] === "instructor-insights" ||
```

Comment:
Does this mean that only a section with the a `short_url` of `instructor-insights` will be process this way? Does this restrict us to never change the page title? 

I think it would be better future-proofed to identify this as a page type instead of by URL.

Reply:
@pdpinch This is a good point and I thought about this.  I had thought that Instructor Insights pages just had a `type` of `CourseSection` (their child pages do) but upon further inspection, the root Instructor Insights page seems to have a `type` of `ThisCourseAtMITSection`.  Does that seem right to you?

Reply:
That's correct. The content type was originally called "This Course at MIT" but then was renamed in the CMS, but not in the code.